### PR TITLE
fix(styles): HCM on form validation icons

### PR DESCRIPTION
**Pull request by leagrdv**

### CODE REVIEW

File: packages/styles/src/functions/_forms.scss [hunk 1]
```diff
@@ -25,3 +30,39 @@
...
+  @return $state;
+}
+
+@function encode-color($color) {
+  @return utilities.replace($color, '#', '%23');
+}
+
+@function valid-icon($color) {
+  $encoded-color: encode-color($color);
+  $icon: utilities.replace(forms.$valid-icon, '%23107800', $encoded-color);
```
Comment (alizedebray) [inline packages/styles/src/functions/_forms.scss:60]: There is a function you can used for that in packages/styles/src/functions/_icons.scss. It is called `add-fill-color`.

**Review by alizedebray — CHANGES_REQUESTED** [review]
The multiple select has a chevron icon that should not be there: 
image

File: packages/styles/src/functions/_forms.scss (hunk 1 above)
Comment (leagrdv) [reply packages/styles/src/functions/_forms.scss:60]: The thing is my variable is not a svg but a `url('...')`. I tried removing the url() wrapper but I get an error in the console telling me it's invalid and missing a `;` and I don't get why. And if I try to use it as it in the fill function, it does not work.

Comment (alizedebray) [reply packages/styles/src/functions/_forms.scss:60]: Ah ok, then no worries I simply wrote that comment in case you didn't know

**Review by alizedebray — APPROVED** [review]

**Pull request merged**